src/App.js

```diff
@@ -12,138 +12,101 @@ import People from './components/People';
+      // loadSelfInfo
+      this.loadData(this.props.self_api, "selfInfo")
+    }
+
+    handleSelect = (eventKey) => {
         this.setState({currentview: eventKey});
-    },
-    selectComponent: function(idx) {
-        switch(idx) {
-        case "write-niceties":
-            $('.dropdown-toggle').text('Write Niceties');
-            $('.dropdown-toggle').append('<span class="caret"></span>');
-            return <People people={this.state.people}
-                            fromMe={this.state.fromMe}
-                            save_nicety_api={this.props.save_nicety_api} />
-        case "view-niceties":
-            $('.dropdown-toggle').text('Niceties About You');
-            $('.dropdown-toggle').append('<span class="caret"></span>');
-            return <NicetyDisplay niceties={this.state.niceties} />
-        case "admin":
-            $('.dropdown-toggle').text('Admin');
-            $('.dropdown-toggle').append('<span class="caret"></span>');
-            return <Admin admin_edit_api={this.props.admin_edit_api}/>
-        default:
+    }
+
+    selectComponent = (idx) => {
+        switch (idx) {
+            case "write-niceties":
+                $('.dropdown-toggle').text('Write Niceties');
+                $('.dropdown-toggle').append('<span class="caret"></span>');
+                return <People
+                    people={this.state.people}
+                    fromMe={this.state.fromMe}
+                    save_nicety_api={this.props.save_nicety_api}/>
+            case "view-niceties":
+                $('.dropdown-toggle').text('Niceties About You');
+                $('.dropdown-toggle').append('<span class="caret"></span>');
+                return <NicetyDisplay niceties={this.state.niceties}/>
+            case "admin":
+                $('.dropdown-toggle').text('Admin');
+                $('.dropdown-toggle').append('<span class="caret"></span>');
+                return <Admin admin_edit_api={this.props.admin_edit_api}/>
+            default:
         }
-    },
-    render: function() {
+    }
+
+    render() {
         let selectedComponent = this.selectComponent(this.state.currentview);
         let adminMenu = null;
         if (this.state.selfInfo.admin === true) {
             adminMenu = (<MenuItem eventKey="admin">Admin</MenuItem>);
         }
         return (
             <div className="App">
-                <Navbar fixedTop id="main_nav">
-                <div id="title">
-                    recurse<br />
-                    nice-<br />
-                    ties
-                </div>
-                <img id="octotie" src={octotie} height="153"/>
+                <Navbar fixedTop="fixedTop" id="main_nav">
+                    <div id="title">
+                        recurse<br/>
+                        nice-<br/>
+                        ties
+                    </div>
+                    <img id="octotie" src={octotie} alt="" height="153"/>
```

Comment:
If we're adding an `alt` attribute, should we also add alt text? :smiley: Maybe something like:
```suggestion
                    <img id="octotie" src={octotie} alt="an octopus wearing a nice tie" height="153"/>
```